File: Facturas_Angular/Facturas/src/app/API/models/linea-factura.ts
/* tslint:disable */
/* eslint-disable */
import { Factura } from './factura';
export interface LineaFactura {
  concepto?: null | string;
  factura?: Factura;
  facturaId?: number;
  importe?: number;
  lineaFacturaId?: number;
  precio?: number;
  unidades?: number;
}

File: Facturas/Models/LineaFactura.cs
namespace Facturas.Models
{
    public class LineaFactura
    {
 
            public int LineaFacturaId { get; set; }
            public int FacturaId { get; set; }
            public string Concepto { get; set; }
            public int Unidades { get; set; }
            public decimal Precio { get; set; }
        public Factura? Factura { get; set; }
        public decimal Importe => Unidades * Precio;
     
        }
    }

